vpr/src/base/vpr_context.h

```diff
@@ -87,89 +100,118 @@ struct hash<std::tuple<int, int, short>> {
 };
 } // namespace std
 
-//State relating the device
-//
-//This should contain only data structures describing the targeted device.
+/**
+ * @brief State relating the device
+ *
+ * This should contain only data structures describing the targeted device.
+ */
 struct DeviceContext : public Context {
     /*********************************************************************
      * Physical FPGA architecture
      *********************************************************************/
-    /* x and y dimensions of the FPGA itself */
-    DeviceGrid grid; /* FPGA complex block grid [0 .. grid.width()-1][0 .. grid.height()-1] */
 
-    /* Empty types */
+    /*
+     * x and y dimensions of the FPGA itself
+     */
```

Comment:
I think this comment is old and can be deleted.

Reply:
I've deleted the comment